src/data_vis.ts

```diff
@@ -261,7 +221,7 @@ class DataVisModel {
         this.renderImpact();
     }
 
-    burdenOutcome = ko.computed(function () {
+    burdenOutcome = ko.computed(() => {
```

Comment:
My Intellisense couldn't recognise what the `this` keyword referred to inside these functions, but changing to lambda syntax means it can which makes life easier. So I've changed all the computed variables to use lambda syntax.